### PR TITLE
Add additional response models in viewsets

### PR DESCRIPTION
**Pull request by usernameMAI**

### CODE REVIEW

File: freddie/viewsets/generics.py [hunk 1]
```diff
@@ -204,6 +204,7 @@ def api_actions(self) -> None:
 
         pk_type = self.pk_type
         status_code = int(HTTPStatus.OK)
+        response_model = getattr(self, 'item_schema', self.schema)  # type: ignore
```
Comment (pysalt) [inline freddie/viewsets/generics.py:207]: for detail view lets use self.schema, this property is changeable in ViewSets

File: freddie/viewsets/generics.py [hunk 2]
```diff
@@ -169,7 +169,7 @@ def get_list_dependencies(self) -> 'PredefinedDependencies':
     def api_actions(self) -> None:
         super().api_actions()
         status_code = int(HTTPStatus.OK)
-        response_model = List[self.schema]  # type: ignore
+        response_model = getattr(self, 'list_schema', List[self.schema])  # type: ignore
```
Comment (pysalt) [inline freddie/viewsets/generics.py:172]: this is a good decision 👍

File: freddie/viewsets/generics.py (hunk 1 above)
Comment (pysalt) [reply freddie/viewsets/generics.py:207]: oh..but schema is used for db and we couldn't change it for response. Lets stay it, but rename to 'detail_schema'

**Comment by pysalt** [comment]
Let's write some tests for new functionality

**Comment by usernameMAI** [comment]
>

File: freddie/viewsets/generics.py (hunk 1 above)
Comment (usernameMAI) [reply freddie/viewsets/generics.py]: added a test that checks that the scheme will match detail or list schema

File: README.md [hunk 3]
```diff
@@ -47,6 +47,7 @@ from fastapi import FastAPI
 from freddie.exceptions import NotFound
 from freddie.schemas import Schema
 from freddie.viewsets import FieldedViewset, PaginatedListViewset, route, ViewSet
+from typing import List
```
Comment (pysalt) [inline README.md:50]: move this import before `from fastapi import FastAPI`

Comment (usernameMAI) [reply README.md]: fixed

**Pull request merged**